x/cronos/types/interfaces.go

```diff
@@ -60,7 +61,8 @@ type EvmLogHandler interface {
 	// Return the id of the log signature it handles
 	EventID() common.Hash
 	// Process the log
-	Handle(ctx sdk.Context, contract common.Address, data []byte, addLogToReceipt func(contractAddress common.Address, logSig common.Hash, logData []byte)) error
+	Handle(ctx sdk.Context, sender common.Address, contract common.Address, data []byte,
```

Comment:
the reason we are adding the field `sender common.Address` is because in the current ERC20.sol, we are not passing msg.sender to `SendToEthereumEvent` https://github.com/crypto-org-chain/cronos/blob/main/contracts/src/ModuleCRC20.sol#L44

While msg.sender is not always equal to tx's from, this is the compromise to avoid migrating all existing CRC20 contract